working-together/README.md

```diff
@@ -245,10 +245,68 @@ We've compiled some guidelines from various sources to help people feel included
 - **Bathrooms**
   - Provide access to all-gender, accessible bathrooms.
 
+## Giving and Receiving Feedback
```

Comment:
One of the goals of adding this section was to provide a way to approach difficult conversations (which could be general, but we were specifically thinking about conversations involving DEI matters and the commitment of all parties to DEI).

While having an open space for honest feedback helps, I wonder if there is anything else I can introduce here explicitly on giving feedback on DEI matters or if what I have is enough.